src/components/kubectl/port-forward.ts

```diff
@@ -234,26 +266,45 @@ function validatePortPair(portPair: string): ValidationResult {
     };
 }
 
+/**
+ * Validates if the port is a named port or withing the valid range
+ * @param validPorts List of valid named ports
+ * @param port The port to validate
+ * @param nullable Can the port be empty or zero
+ * @returns Boolean identifying if the port is valid
+ */
+function isPortValid(validPorts: ExtractedPort[], port: string, nullable?: boolean): boolean {
```

Comment:
I would like the `nullable` argument to be mandatory because it's not obvious which way the default would be.  Even better might be to make it an enum e.g. `PortSpecifier.{ Required | AllowEmpty }` as this will be more self-documenting at the call site.